site/styles/_buttons.scss

```diff
@@ -115,8 +115,8 @@
     background: transparent;
 
     .s72-icon-play circle {
-      fill: $primary;
-      stroke: $primary;
+      fill: var(--primary);
```

Comment:
should these not be `var(--primary-rgb)`?

Reply:
Ah --primary is set based off --primary-rgb `--primary: rgb(var(--primary-rgb));`